Dockerfile

```diff
@@ -1,37 +1,33 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
 ARG DEBIAN_FRONTEND=noninteractive
-MAINTAINER https://github.com/huard/raven
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="https://github.com/Ouranosinc/raven"
 LABEL Description="Raven WPS" Vendor="Birdhouse" Version="0.18.2"
 
-# Update Debian system
-RUN apt-get update && apt-get install -y \
- build-essential unzip libnetcdf-dev curl \
```

Comment:
By the way, we don't need to install additional debian packages anymore?  Did you annotate to see why they were needed in the past?